### DISCUSSION TITLE
How to set unit "em" or "rem" for font sizes ?

### DISCUSSION THREAD

**Discussion by jchome** [xdan/jodit#1057] [Q&A] [answered]
Hi, In documentation, I've seen that it's possible to set font sizes, with an array of integers. In source code, there is a "defaultFontSizePoints", that takes "px" or "pt".

But how can I set the font size unit to "em" or "rem" ?

**Comment by xdan** [comment, marked as answer]
The fontsize control works in px (or pt if you set `defaultFontSizePoints: 'pt'`) — em/rem isn't a built-in unit option. If you need em/rem you'd apply it yourself: either a custom control that sets `font-size` with your unit, or post-process the output. There's no config flag to switch the built-in fontsize list over to em/rem.
